producers/kubernetes/ingress.go

```diff
@@ -57,8 +57,8 @@ func (a *kubernetesIngressProducer) Endpoints() ([]*pkg.Endpoint, error) {
 
 		eps, err := a.convertIngressToEndpoint(ing)
```

Comment:
instead of ignoring services and ingresses with multiple external addresses it processes the first one found and prints a warning (because I've never seen this case in a real cluster).